server/php-engine.cpp

```diff
@@ -1873,7 +1877,7 @@ int main_args_handler(int i, const char *long_option) {
       return vk::singleton<ServerConfig>::get().init_from_config(optarg);
     }
     case 't': {
-      script_timeout = static_cast<int>(normalize_script_timeout(atoi(optarg)));
+      script_timeout = static_cast<int>(normalize_script_timeout(parse_time_limit(optarg)));
```

Comment:
Let's check that default specifier for `parse_time_limit` is seconds

Reply:
I checked it out
--time-limit 5 and --time-limit 5s work the same way